src/utils/validators.ts

```diff
@@ -55,11 +55,18 @@ export const maxParticipantsValidator = (
 };
 
 export const nameValidator = (name: string): string[] | undefined => {
-  const nameReg = /^[a-zæøåA-ZÆØÅ ]+$/; //! note the last space
+  // \p{L} -> letter in any language
+  // \p{Zz} -> allows space
+  // 1,30 is the valid range
+  // must be an range or any name over one char gets an error
+  const nameReg = /^[\p{L}\p{Zs}]{1,30}$/u;
 
   if (name.length === 0) {
     return ['Navn er påkrevd'];
   }
+  if (name.length > 30) {
+    return ['Navnet er for langt, maks 30 bokstaver'];
+  }
   if (!new RegExp(nameReg).test(name)) {
     return ['Kun bokstavene A-Å'];
```

Comment:
Could maybe give a different message here, since we allow more than A-Å now

Reply:
whoops, forgot about that one